scripting/get5.sp

```diff
@@ -1089,13 +1089,28 @@ static Action Command_LoadMatchUrl(int client, int args) {
     ReplyToCommand(client,
                    "Cannot load matches from a url without the SteamWorks extension running");
   } else {
-    char arg[PLATFORM_MAX_PATH];
-    if (args >= 1 && GetCmdArgString(arg, sizeof(arg))) {
-      if (!LoadMatchFromUrl(arg)) {
+    char url[PLATFORM_MAX_PATH];
+    if (args >= 1 && GetCmdArg(1, url, sizeof(url))) {
+      ArrayList headerNames;
+      ArrayList headerValues;
+      if (args == 2) {
+        ReplyToCommand(client, "This command accepts either one or three parameters.");
+        return;
+      }
+      if (args == 3) {
+        char headerbuffer[PLATFORM_MAX_PATH];
+        GetCmdArg(2, headerbuffer, sizeof(headerbuffer));
+        headerNames.PushString(headerbuffer);
```

Comment:
`headerNames` and `headerValues` are now `null`, as you declared them but did not create the lists (`new ArrayList...`). Move creation of both lists inside the `if (args == 3)`-check.

Reply:
@nickdnk done